### DISCUSSION TITLE
Create multi graph elements with api return 404（/graphs/hugegraph/hugegraph_test/vertices/batch）

### DISCUSSION THREAD

**Discussion by hwq820** [apache/incubator-hugegraph#2317] [Q&A] [answered]
### Problem Type (问题类型)

None

### Before submit

- [X] 我已经确认现有的 [Issues](https://github.com/apache/hugegraph/issues) 与 [FAQ](https://hugegraph.apache.org/docs/guides/faq/) 中没有相同 / 重复问题 (I have confirmed and searched that there are no similar problems in the historical issue and documents)

### Environment (环境信息)

- Server Version: 1.0.0 (Apache Release Version)
- Backend: RocksDB x nodes, HDD or SSD 
- OS: xx CPUs, xx G RAM, Ubuntu 2x.x / CentOS 7.x 
- Data Size:  xx vertices, xx edges 

### Your Question (问题描述)

{
    "exception": "class javax.ws.rs.NotFoundException",
    "message": "HTTP 404 Not Found",
    "cause": ""
}

**Comment by liuxiaocs7** [comment]
Hi, @hwq820, you can refer this PR https://github.com/apache/incubator-hugegraph-doc/pull/259 and doc https://hugegraph.apache.org/cn/docs/clients/restful-api/vertex/ here firstly.

And could you provide a more detailed description?

**Comment by hwq820** [comment]
> Hi, @hwq820, you can refer this PR [apache/incubator-hugegraph-doc#259](https://github.com/apache/incubator-hugegraph-doc/pull/259) and doc https://hugegraph.apache.org/cn/docs/clients/restful-api/vertex/ here firstly.
> 
> And could you provide a more detailed description?
路径是按照文档的路径拼写的，hugegraph_test是我的图库的名称，调用的主机端口都没问题

**Comment by liuxiaocs7** [comment, marked as answer]
you mean this: `/graphs/hugegraph/hugegraph_test/vertices/batch`?
try `/graphs/hugegraph_test/graph/vertices/batch` instead, where `hugegraph_test` is your graph name
